src/auth_identity.rs

```diff
@@ -260,6 +261,8 @@ mod scard_credentials {
     pub struct SmartCardIdentity {
         /// Username
         pub username: String,
+        /// Domain
+        pub domain: String,
```

Comment:
Why do you need a domain here? usually, the domain is extracted from the FQDN username or user certificate.